daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -1482,6 +1518,74 @@ object Main {
         }
       }
 
+      case Some(conf.encodeEXI) => {
+        val encodeOpts = conf.encodeEXI
```

Comment:
I'm puzzled why we're seeing codecov check warnings that some lines are not covered by tests, when we do have a new test.